resources/lang/pl_PL/models.yml

```diff
@@ -0,0 +1,57 @@
+generic:
+  createdAt: "utworzono"
```

Comment:
Jak już chciałeś YAMLa to może całkiem bez cudzysłowów?

Reply:
przynajmniej kiedyś phpstorm sprawdzał pisownię tylko jak było w cudzysłowie

Reply:
i no jakoś wolę tak